charts/fybrik/files/taxonomy/data_catalog_response.json

```diff
@@ -0,0 +1,82 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Data Catalog Response",
+  "type": "object",
+  "properties": {
+    "name": {
+      "$ref": "#/definitions/assetName"
+    },
+    "operationType": {
+      "$ref": "taxonomy.json#/definitions/OperationType"
+    },
+    "asset": {
+      "$ref": "#/definitions/assetSpec"
+    }
+  },
+  "required": [
+    "name",
+    "asset"
+  ],
+  "additionalProperties": false,
+  "definitions": {
+    "assetName": {
+      "type": "string",
+      "description": "Name of the asset"
+    },
+    "asset": {
+      "$ref": "#/definitions/assetSpec"
+    },
+    "assetSpec": {
+      "type": "object",
+      "properties": {
+        "assetCredentials": {
+          "$ref": "#/definitions/vaultPluginPath"
+        },
+        "assetMetadata": {
+          "$ref": "#/definitions/assetMetadata"
+        },
+        "assetDetails": {
+          "$ref": "#/definitions/assetDetails"
+        }
+      },
+      "required": [
+        "assetCredentials",
+        "assetMetadata",
+        "assetDetails"
+      ]
+    },
+    "vaultPluginPath": {
+      "type": "string",
+      "description": "This has the vault plugin path where the data credentials will be stored as kubernetes secrets. This value is assumed to be known to the catalog connector."
+    },
+    "assetDetails": {
+      "type": "object",
+      "description": "Asset details",
+      "properties": {
+        "connection": {
+          "$ref": "taxonomy.json#/definitions/Connection"
+        },
+        "dataFormat": {
+          "$ref": "taxonomy.json#/definitions/Dataformat"
+        }
+      },
+      "required": [
+        "connection"
+      ]
+    },
+    "assetMetadata": {
+      "type": "object",
+      "properties": {
+        "owner": {
+          "type": "string"
+        },
+        "geography": {
+          "type": "string"
+        },
+        "metadata": {
+          "$ref":"policy_manager_request.json#/definitions/Resource"
```

Comment:
This should be owned by the catalog not the policy manager. I suggest to add data_catalog.json with common definitions like that. Should probably also rename Resource accordingly.

Reply:
I have added data_catalog.json to have Resource definition. However change the name "Resource" to something else would result in generated code for policy manager to change which may have other dependencies. Can we postpone the renaming to another PR at a later stage ? Please review. Thanks.